cmd/tealdbg/debugger.go

```diff
@@ -187,6 +188,28 @@ func (s *session) Step() {
 	s.resume()
 }
 
+func (s *session) StepOver() {
+	currentLine := s.line.Load()
+	// Get the first TEAL opcode in the line
+	currentOp := strings.Fields(s.lines[currentLine])[0]
+
+	func() {
+		s.mu.Lock()
+		defer s.mu.Unlock()
+		// Step over a function call (callsub op).
+		if currentOp == "callsub" {
+			err := s.setBreakpoint(currentLine + 1)
```

Comment:
I think the only time it would error would be if `callsub` was at the end of the program and `setBreakpoint()` will try to set a breakpoint past the program. In that case, it feels normal to not log something and just continue, but maybe I'm missing a case.